### ISSUE TITLE
DeGoogled Purchases

### ISSUE THREAD

**Issue by xalgia** [BranchMetrics/sesame_issues#247]
Please implement a way to purchase this app outside of the Google Play Store. There are many of us who love this app but refuse to be abused by companies like google by using degoogled AOSP ROMs with no play store to verify our purchases.

More and more are waking up everyday to the dangers of centralised tech and tyrannical data collection and it will be in your best interests to future proof your app (while at the same time making it accessible to a much larger audience) by implementing an independent purchase and update channel.

**Comment by gabrc52** [comment]
Agreed. It's impossible to use Sesame without official Google services.
